### PR TITLE
BatchRequest args fix: need toString operation for different data types

### PR DESCRIPTION
**Pull request by lightning-L**
### _Why are the changes needed?_

Refine the CreateBatchCommand.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala [hunk 1]
```diff
@@ -43,13 +43,15 @@ class CreateBatchCommand(cliConfig: CliConfig) extends Command[Batch](cliConfig)
       val request = map.get("request").asInstanceOf[JMap[String, Object]]
       val config = request.get("configs").asInstanceOf[JMap[Object, Object]].asScala
         .map { case (k, v) => (k.toString, v.toString) }.asJava
+      val args = request.get("args").asInstanceOf[ArrayList[Object]].asScala
```
Comment (turboFei) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:46]: ArrayList => JList

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala [hunk 2]
```diff
@@ -43,13 +43,15 @@ class CreateBatchCommand(cliConfig: CliConfig) extends Command[Batch](cliConfig)
...
+        .map(x => x.toString).asJava
       val batchRequest = new BatchRequest(
         map.get("batchType").asInstanceOf[String],
```
Comment (turboFei) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: should the  batchType from request？

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]:
```
request.get("batchType").asInstanceOf[String]
```

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: BTW, the batchType in https://github.com/apache/incubator-kyuubi/blob/master/kyuubi-ctl/src/test/resources/cli/batch.yaml is missing

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: image

cc @ulysses-you

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: how about read from `map` first, if not defined, then fall back to `request`.

Comment (ulysses-you) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: how can we fall back to request, do you mean  ?
```yml
request:
  name:
  batchType:
```

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:49]: yes.

Because it is a filed of dto `BatchRequest`.

Comment (ulysses-you) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala:48]: seems we do not need read from map first, just get it from request

**Review by turboFei — APPROVED** [review]
LGTM

**Review by ulysses-you — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**